SUMMARY.md

```diff
@@ -124,6 +124,7 @@
 * [Conventions](conventions/README.md)
    * [Coding style](conventions/coding_style.md)
    * [Documenting code](conventions/documenting_code.md)
+   * [Writing Shards](conventions/writing_shards.md)
```

Comment:
Can we move it to the `Guides` section? I think it fits better there

Reply:
Sure! I just went with `Conventions` because the `Conventions` section seemed to be more focused on the soft skills, per se. Especially, making code "more accessible to other developers". Whereas the `Guides` were more hard skills centric, with the `Performance` and `Concurrency` sections. But it certainly _is_ a guide! Update is on the way